traffic_ops/testing/api/v3/topologies_test.go

```diff
@@ -240,6 +241,52 @@ func UpdateTestTopologies(t *testing.T) {
 	}
 }
 
+func UpdateValidateTopologyORGServerCacheGroup(t *testing.T) {
+	params := url.Values{}
+	params.Set("xmlId", "ds-top")
+
+	//Get the correct DS
+	remoteDS, _, err := TOSession.GetDeliveryServicesV30WithHdr(nil, params)
+	if err != nil {
+		t.Errorf("cannot GET Delivery Services: %v", err)
+	}
+
+	//Assign ORG server to DS
+	assignServer := []string{"denver-mso-org-01"}
+	_, _, err = TOSession.AssignServersToDeliveryService(assignServer, *remoteDS[0].XMLID)
+	if err != nil {
+		t.Errorf("cannot assign server to Delivery Services: %v", err)
+	}
+
+	//Get Topology node to update and remove ORG server nodes
+	origTopo := *remoteDS[0].Topology
+	resp, _, err := TOSession.GetTopologyWithHdr(origTopo, nil)
+	if err != nil {
+		t.Fatalf("couldn't find any topologies: %v", err)
+	}
+
+	// remove org server cachegroup
+	var p []int
+	newNodes := []tc.TopologyNode{{Id: 0, Cachegroup: "topology-edge-cg-01", Parents: p, LastUpdated: nil}}
+	if *remoteDS[0].Topology == resp.Name {
+		resp.Nodes = newNodes
+	}
+	_, _, err = TOSession.UpdateTopology(*remoteDS[0].Topology, *resp)
+	if err == nil {
+		t.Fatalf("shouldnot UPDATE topology:%v to %v, but update was a success", *remoteDS[0].Topology, newNodes[0].Cachegroup)
+	}
```

Comment:
Because I ran into a test issue like this the other day, we should add an `else if` to check that the error message contains `"ORG servers are assigned to delivery services that use this topology, and their cachegroups cannot be removed"`.

We want to make sure we're testing _that_ validation specifically, and that the topology update would otherwise be successful if not for that specific validation.